### DISCUSSION TITLE
Feature Request: Scheduled data source refresh

### DISCUSSION THREAD

**Discussion by kevev** [Budibase/budibase#1609] [Ideas]
A killer feature to add would be an "Auto-Refresh Timer" field added to the data source configuration. This would allow automatic data source refreshes. This feature would allow for creation of Live Dashboards for monitoring.

**Comment by aptkingston** [comment]
Thanks for the suggestion @kevev. This could definitely make sense for applications like live dashboards and isn't hard to implement.

**Comment by joebudi** [comment]
I am converting this to a discussion, and labeling idea 👍

**Reply by joebudi** [reply]
Conversion complete.

**Comment by stlsw** [comment]
This is really a turndown feature, not only for dashboard, but data auditing by support personnel.  is there a workaround way to refresh it ? as of now, we will be continue to use Google Data Studio.

**Reply by aptkingston** [reply]
What sort of refresh interval would be ideal for your use case? It would probably be configurable (in seconds) as a setting on the data provider component, but I'm just curious.

**Reply by stlsw** [reply]
having configurable data refresh function on data provider component will be great. Even manual button to refresh data will satisfy our scenario.

**Comment by aptkingston** [comment]
You can track this here: https://github.com/Budibase/budibase/issues/2427.

**Reply by stlsw** [reply]
Wow. thank you ! looking forward to
